website/src/menu.css.ts

```diff
@@ -10,6 +10,7 @@ import sprinkles, {
   vspace,
 } from "src/sprinkles.css"
 import { paddingX, paddingY } from "src/style-utils"
+import * as cwkwTheme from "./cwkw/theme.css"
```

Comment:
Never use themes directly, always use the variables from the theme contract defined in the `sprinkles.css` file. That way if you switch themes, the styles will update accordingly.